### PR TITLE
Update docs pathing

### PR DESCRIPTION
**Pull request by chrismaree**
This PR fixes a bunch of issues due to the most recent lerna re-structuring.

### CODE REVIEW

**Comment by adrianmcli** [comment]
lgtm

File: docs/tutorials/bots.md [hunk 1]
```diff
@@ -93,17 +93,16 @@ If you are not using Linux (or if you have installed the above), you can run the
...
+Compile contracts
+npx lerna bootstrap
 
-# Navigate into the core directory & compile contracts
-cd ./core
-npx truffle compile
 ```
 
-If you have any issues executing `npx truffle <command>` you can try running `$(npm bin)/truffle <command>`. If you have Truffle globally installed you should be able to run `truffle <command>` without npx or `$(npm bin)` but it may not be the same version specified in the project.
+If you have any issues executing `yarn truffle <command>` you can try running `$(npm bin)/truffle <command>`. If you have Truffle globally installed you should be able to run `truffle <command>` without npx or `$(npm bin)` but it may not be the same version specified in the project. The liquidator and disputer scripts can also be run directly as node processes by replacing the `yarn truffle exec <path>` commands with `node <path>`.
```
Comment (mrice32) [inline docs/tutorials/bots.md:105]: Nice!

File: docs/tutorials/bots.md [hunk 2]
```diff
@@ -119,7 +118,7 @@ All deployment configurations require a wallet mnemonic (or private key) to be i
 You can either bring your own mnemonic from an existing wallet or generate a fresh one using the `bip39` package installed within the UMA repo.
 If you have a wallet mnemonic already you can skip this section.
 
-To generate a new mnemonic you can run the following from the `/core` directory:
+To generate a new mnemonic you can run the following :
```
Comment (mrice32) [inline docs/tutorials/bots.md:121]:
```suggestion
To generate a new mnemonic you can run the following:
```

File: docs/tutorials/cli-tool.md [hunk 3]
```diff
@@ -46,17 +46,17 @@ You may need to prefix these commands (`sudo npm link`) to run them.
...
 ## Launching the CLI tool
 
-1. Navigate to the `/core` folder: `cd core`
+1. Navigate to the `packages/core` folder: `cd packages/core`
 2. If on Kovan testnet, apply network addresses:
 
 ```bash
-npx apply-registry
+yarn load-addresse
 ```
```
Comment (mrice32) [inline docs/tutorials/cli-tool.md:54]: nit: if we're assuming they've already compiled the contracts using `qbuild`, the contracts should already be pre-loaded with the kovan addresses -- so I _think_ this means we can entirely remove this step.

File: docs/tutorials/cli-tool.md [hunk 4]
```diff
@@ -46,17 +46,17 @@ You may need to prefix these commands (`sudo npm link`) to run them.
 
 ## Launching the CLI tool
 
-1. Navigate to the `/core` folder: `cd core`
+1. Navigate to the `packages/core` folder: `cd packages/core`
```
Comment (mrice32) [inline docs/tutorials/cli-tool.md:49]: Can all of the steps below be done from root?

File: docs/tutorials/dvm-integration.md [hunk 5]
```diff
@@ -33,13 +33,13 @@ Details on these two fees are available [here](oracle/econ-architecture.md#step-
 2. Migrate the contracts by running the following command:
 
 ```bash
-$(npm bin)/truffle migrate --reset --network test
+yarn truffle migrate --reset --network test
 ```
 
-3. To deploy the `DepositBox` contract and go through a simple user flow, run the following script from the `/core` directory:
+3. To deploy the `DepositBox` contract and go through a simple user flow, run the following script from the `packages/core` directory:
```
Comment (mrice32) [inline docs/tutorials/dvm-integration.md:39]: nit: I think these commands can all be run from root.

File: docs/tutorials/mint-locally.md [hunk 6]
```diff
@@ -7,7 +7,7 @@ This tutorial will show you how to create synthetic tokens from the command line
 
 - Have the protocol repo cloned.
 - Be running an instance of Ganache on port 9545.
-- Have run truffle compilation for the contracts in `core/`.
+- Have run truffle compilation for the contracts in `packages/core` or have run `npx lerna run build`.
```
Comment (mrice32) [inline docs/tutorials/mint-locally.md:10]:
```suggestion
- Have installed dependencies using `yarn` and have run `yarn qbuild` to build the contracts.
```

File: docs/tutorials/setup.md [hunk 7]
```diff
@@ -12,13 +12,16 @@ The following steps require the `git` CLI to be installed. If you are on Windows
 Clone the UMA [repo](https://github.com/UMAprotocol/protocol). Start in the top-level directory in this repository, `protocol/`.
 
 1. Install the latest stable version of [Node.js](https://nodejs.org/) and [Yarn](https://classic.yarnpkg.com/) is installed along with it.
-2. Run `yarn` in `protocol/`.
+2. Run the following in the root of the repo to install all packages from the UMA mono repo:
+```bash
+npx lerna bootstrap
```
Comment (mrice32) [inline docs/tutorials/setup.md:17]:
```suggestion
yarn
```

File: docs/tutorials/setup.md [hunk 8]
```diff
@@ -12,13 +12,16 @@ The following steps require the `git` CLI to be installed. If you are on Windows
...
+```
+
 
-We should be able to compile the smart contracts from `protocol/core`:
+We should be able to compile the smart contracts:
 
 ```bash
-cd core
-npx truffle compile
+npx lerna run build
```
Comment (mrice32) [inline docs/tutorials/setup.md:24]:
```suggestion
yarn qbuild
```

File: docs/tutorials/setup.md [hunk 9]
```diff
@@ -39,11 +42,11 @@ Here, we are running Ganache:
 
 This will make sure we have enough ETH and a high enough gas limit to deploy our larger contracts.
 
-If everything was setup correctly, we should be able to run automated tests from `protocol/core`. In a separate terminal, run the following commands from the `protocol/core` directory:
+If everything was setup correctly, we should be able to run automated tests from `protocol/packages/core`. In a separate terminal, run the following commands from the `protocol/packages/core` directory:
```
Comment (mrice32) [inline docs/tutorials/setup.md:45]: nit: I know this was already phrased this way, but it's kinda weird that we're telling them to run `cd packages/core` _from_ the `packages/core` directory. I feel like we should either remove the `cd` or tell them to run the commands from root.

File: docs/tutorials/bots.md [hunk 10]
```diff
@@ -93,17 +93,16 @@ If you are not using Linux (or if you have installed the above), you can run the
 ```bash
 # Clone the repo and navigate into the protocol directory
 git clone https://github.com/UMAprotocol/protocol.git
-cd ./protocol
```
[the remark this answers is not in the document]
Comment (chrismaree) [reply docs/tutorials/bots.md:96]: good catch. should not be. have reverted it

File: docs/tutorials/bots.md [hunk 11]
```diff
@@ -93,17 +93,16 @@ If you are not using Linux (or if you have installed the above), you can run the
...
 
 # Install dependencies
-yarn
+npx lerna bootstrap
+
+Compile contracts
+npx lerna bootstrap
```
[the remark this answers is not in the document]
Comment (chrismaree) [reply docs/tutorials/bots.md:101]: ok cool. I think that's fine, yes?

File: docs/tutorials/cli-tool.md (hunk 4 above)
Comment (chrismaree) [reply docs/tutorials/cli-tool.md:49]: ye, I think they should be able to be run from root. I've refactored it to reflect this. take a look & LMK.

File: docs/tutorials/setup.md (hunk 9 above)
Comment (chrismaree) [reply docs/tutorials/setup.md]: yup. I agree. Seeing the code steps includes a `cd packages/core` I removed this from the main sentence. Should flow a bit more easily now.

File: docs/tutorials/cli-tool.md [hunk 12]
```diff
@@ -28,7 +28,7 @@ Before beginning this tutorial, please make sure your environment is set up corr
 
 - Have the protocol repo cloned.
 - Be running an instance of Ganache on port 9545.
-- Have run truffle compilation for the contracts in `core/`.
+- Have run truffle compilation for the contracts in `packages/core` or have run `npx lerna run build`. If you've run `yarn qbuild` you've done this already.
```
Comment (mrice32) [inline docs/tutorials/cli-tool.md:31]:
```suggestion
- Have run truffle compilation for the contracts in `packages/core`. If you've run `yarn qbuild` you've done this already.
```

File: docs/tutorials/dvm-integration.md (hunk 5 above)
Comment (mrice32) [reply docs/tutorials/dvm-integration.md:39]: bump

Comment (chrismaree) [reply docs/tutorials/dvm-integration.md]: done.

**Pull request merged**